### DISCUSSION TITLE
Are you open to Design contributions?

### DISCUSSION THREAD

**Discussion by fivenp** [mastodon/mastodon#21967] [Q&A]
I know you all are for sure VERY busy these days but I have a very serious question for you folks :) 

I'm currently trying to push an initiative to foster and enable Designers to contribute to Open Source software: 
https://contribute.design

Now I'm curious if you'd be up to participate in this initiative and welcome design contributions to mastodon? If so I'd be very glad to help you (with a PR) creating a first draft of design guidelines and redirect designers towards your project.

**Comment by b-d-m-p** [comment]
Wondering the same thing. I am happy to help also.

**Reply by fivenp** [reply]
Maybe we can start by defining a first draft of proposed design contribution processes? 

And then start documenting where to find resources, documentation for past decisions, etc etc

Will all be quite hard without the core maintainers involved in the initial draft phase though I guess....

**Comment by fivenp** [comment]
I did a first stab at having a basic set of contribution guidelines.... Maybe they get merged and we all know a little bit better where / how to get things rolling :) 

#22100
